### PR TITLE
feat(postcard_create): Create an Angular service for creating a postcard

### PR DESCRIPTION
**Pull request by vinay-pad**
What: Create an angular service for create a postcard and writing unit test to test the service.

Why: Part of the miyagi angular bootcamp.

Details:

Created a new postcard service that provides a create() method.
Unit test to check if the request arguments and payload are correct.

### CODE REVIEW

File: app/services/index.js [hunk 1]
```diff
@@ -5,12 +5,14 @@ var Cookies = require('angular-cookies');
 
 var API     = require('./api');
 var Auth    = require('./auth');
+var Postcard = require('./postcard');
```
Comment (thetonywu) [inline app/services/index.js]: Nit: align equal signs

**Review by thetonywu — COMMENTED** [review]
I think you need to go into Travis and enable it for your project

File: test/services/postcard.spec.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+var expect  = require('chai').expect;
+var Angular = require('angular');
+var Config  = require('../../config');
```
Comment (thetonywu) [inline test/services/postcard.spec.js:5]: nit: alpha order

also we normally separate imports from external packages and code from the same repo into different sections separated by a blank line, so you can move `var Config  = require('../../config');` below these

**Review by thetonywu — APPROVED** [review]
LGTM

**Comment by vinay-pad** [comment]
@tonywuhl Is one LGTM good for this or should i assign it to someone else?

**Comment by thetonywu** [comment]
let's merge it!

**Pull request merged**